lib/context_adapter.js

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+ *
+ * This file is part of context-adapter
+ *
+ * context-adapter is free software: you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * context-adapter is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public
+ * License along with context-adapter.
+ * If not, seehttp://www.gnu.org/licenses/.
+ *
+ * For those usages not covered by the GNU Affero General Public License
+ * please contact with::[german.torodelvalle@telefonica.com]
+ */
+
+(function() {
```

Comment:
Agreed! I was using this structure as recommended by Mozilla for Firefox OS applications :D since we had mechanisms to use Node packages from the client-side (Browserify and alike) to avoid `'use strict';` conflicts :) but no longer the case, so removing the IIFE from the files... :) Thanks!